src/job/job.py

```diff
@@ -887,6 +887,14 @@ def run_docker_container(self, image_name):
             if self.job['definition'].get('cache', {}).get('image', False) or self.job['definition'].get('deployments', None):
                 c.execute(("docker", "commit", container_name, image_name))
         except Exception as e:
+            try:
+                # Find out if container build was failed
+                out = subprocess.check_output(['docker', 'images', '-q', image_name]).strip()
+                if not out:
+                    raise Failure("Could not build image of container")
```

Comment:
Maybe we can simply show an error message: `Error running container`

Reply:
OK I have simplied it.